### DISCUSSION TITLE
Surpassed the Within 3 Days for benefits.

### DISCUSSION THREAD

**Discussion by gml-pz** [community/community#145604] [GitHub Education]
### Hi everyone,

Hey I recently applied for the Github Education on Friday night and I have yet to receive my benefits or an update.

Hope you're having a great week so far.

**Comment by davevad93** [comment]
Hi @aljndaro ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @aljndaro was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by andrew8238** [comment]
The waiting time for response on GitHub may vary based on factors like the volume of requests, the complexity of the issue, and current workload. Check [reasons](https://funeralhomesamarillo.us/) for delays, such as ongoing system maintenance or high ticket volumes. Typically, it takes a few days for support to respond, but in some cases, it could take a bit longer, even up to a couple of weeks. Patience is appreciated, and you can check GitHub's status page or community forums for updates.
